.changeset/ninety-turtles-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+fix routing and config for user-settings plugin
```

Comment:
OK two final asks.

First: You need to rebase on the most recent master; we mistakenly caused a build error in master yesterday and it's fixed now.

Second: When doing app template changes, it's nice to add a few lines here (that end up in the changelog) that inform users how to make the corresponding change in their create-app version of the app. Something like

To make the corresponding change in your local app, add the following in your App.tsx:

```ts
import { Router as SettingsRouter } from '@backstage/plugin-user-settings';
...
<Route path="/settings" element={<SettingsRouter />} />
```

and the following to your plugins.ts:

```ts
export { plugin as UserSettings } from '@backstage/plugin-user-settings';
```

Reply:
@freben its done.